content/first-steps.md

```diff
@@ -112,3 +112,49 @@ $ npm run start:dev
 ```
 
 This command will watch your files, automatically recompiling and reloading the server.
+
+#### Linting and formatting
+
+[Nest CLI](/cli/overview) provides best effort to scaffold a reliable development workflow at scale. Thus, a generated Nest project comes with both a code **linter** and **formatter** preinstalled (respectively [eslint](https://eslint.org/) and [prettier](https://prettier.io/)).
+
+> info **Hint** Not sure about the role of formatters VS linters ? Learn the difference [here](https://prettier.io/docs/en/comparison.html).
+
+##### VSCode usage
+To ensure maximum stability and extensibility, we use official [`eslint`](https://www.npmjs.com/package/eslint) and [`prettier`](https://www.npmjs.com/package/prettier) cli packages. This setup allows neat IDE integration with official tooling.
+
+For example with *VSCode*, a Nest project is compatible with [`vscode-eslint`](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [`prettier-vscode`](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) by design !
+
+> warning **Warning** You might be tempted to try [~~`vs-code-prettier-eslint`~~](https://marketplace.visualstudio.com/items?itemName=rvest.vs-code-prettier-eslint). This extension is meant to play with [~~`prettier-eslint`~~](https://github.com/prettier/prettier-eslint) package witch is **not** used inside a Nest project by default.
```

Comment:
Not sure if mentioning any IDE makes sense. I believe it may introduce some confusion, with people thinking that NestJS _recommends_ (or suggests, or simply "integrates best with") using VSCode.

Reply:
Good point. Though, despite the fact Nest is not tied to an IDE by any means, its arguably a fact that VSCode is used by a large majority of developers. The part of extensions suggestions ("*For example with VSCode...*" would probably save time for a majority of developers).

I propose
- Renaming paragraph title for agnostism
- Keeping the paragraph "for example"
- Removing the warning which seems a bit too much here

Reply:
I still don't think that mentioning IDE-specific extensions in the framework's documentation is a reasonable decision though. If we go down this path, we'd have to create a dedicated page specifically for IDEs and provide docs for the most commonly used IDEs, not only VSCode (which on the other hand brings another "maintenance burden" for us and that's probably not what we want).

Reply:
Alright. Just going to keep the paragraph in an agnostic way, mention neat "IDE integration" (agnostic) and mention the fact we're not using `eslint-prettier` (which might save research time).

All in all, people should understand which extension goes with which package; and Nestjs is providing great native support by design.

Reply:
Does that feel OK like this ?